libs/designsystem/src/lib/components/modal/modal-wrapper/modal-wrapper.component.scss

```
@@ -111,6 +112,10 @@ ion-content {
   --padding-end: #{size('s')};
 }
 
+button {
+  color: var(--kirby-modal-color);
```

Consider adding fallback:
```suggestion
  color: var(--kirby-modal-color, #{get-color('black')});
```